src/main/sensors/battery.c

```diff
@@ -92,7 +92,7 @@ void updateBattery(void)
     uint16_t vbatMeasured = batteryAdcToVoltage(vbatLatestADC);
 
     /* battery has just been connected*/
-    if (batteryState == BATTERY_NOT_PRESENT && (ARMING_FLAG(ARMED) || (vbat > batteryConfig->batterynotpresentlevel && ABS(vbatMeasured - batteryAdcToVoltage(vbatPreviousADC)) <= VBAT_STABLE_MAX_DELTA))) {
+    if (batteryState == BATTERY_NOT_PRESENT && vbat > batteryConfig->batterynotpresentlevel && ABS(vbatMeasured - batteryAdcToVoltage(vbatPreviousADC)) <= VBAT_STABLE_MAX_DELTA) {
```

Comment:
Huh? So you'd rather not have the battery warnings when you don't have a battery connected and arm the craft, and risk not getting the warnings when a battery is actually connected and you're flying, just because the voltage is too low? That does not make much sense.